solana/program_v2/packages/tests/src/gatekeeper-suite/create-gatekeeper.test.ts

```diff
@@ -123,6 +107,48 @@ describe('Gateway v2 Client', () => {
           .rpc()
       ).to.eventually.be.rejectedWith(/InsufficientAuthKeys/);
     }).timeout(10000);
+
+    it('Create a gatekeeper with applied token fees', async function () {
+      const tokenKey = Keypair.generate();
+
+      // Step 1: Create a gatekeeper
+      await networkService
+        .createGatekeeper(
+          networkAuthority.publicKey,
+          stakingDataAccount,
+          adminAuthority.publicKey,
+          {
+            tokenFees: [
+              {
+                token: tokenKey.publicKey,
+                // TODO: Fix the representation in: https://github.com/identity-com/on-chain-identity-gateway/blob/develop/solana/program_v2/packages/client/core/src/lib/types.ts#L18-L18
+                // TODO: These parameters are u64 in Rust and cannot be represented as numbers: https://github.com/identity-com/on-chain-identity-gateway/blob/develop/solana/program_v2/programs/gateway_v2/src/state/gatekeeper.rs#L235-L235
+                issue: new BN(0) as unknown as number,
```

Comment:
The idea here was to pass it in as JS numbers, and convert it to BN internally... though, debateable I guess as that could mean we limited to u32 on the JS side?

Reply:
Could we expose bigNumber and not the anchor BN?